crates/stdx/src/lib.rs

```diff
@@ -10,15 +10,15 @@ pub fn is_ci() -> bool {
 
 pub trait SepBy: Sized {
     /// Returns an `impl fmt::Display`, which joins elements via a separator.
-    fn sep_by<'a>(self, sep: &'a str) -> SepByBuilder<'a, Self>;
+    fn sep_by(self, sep: &str) -> SepByBuilder<'_, Self>;
```

Comment:
@matklad what about this? IMHO, anonymous lifetime looks a bit more aesthetical.